llvm-bazel/llvm-project-overlay/llvm/tblgen.bzl

```
@@ -35,6 +35,9 @@ def gentbl(
       tblgen_args: Extra arguments string to pass to the tblgen binary.
       **kwargs: Keyword arguments to pass to subsidiary cc_library() rule.
     """
+    llvm_project_workspace = Label("//:...", relative_to_caller_repository = False)
+    llvm_project_execroot_path = llvm_project_workspace.workspace_root
```

I think this isn't really a well-formed label. What about using this file itself. We know that exists :-)

```suggestion
    llvm_project_execroot_path = Label("//llvm:tblgen.bzl", relative_to_caller_repository = False).workspace_root
```

---

I don't think it _actually_ matters since we never use the label itself anywhere but sure 👍

---

Yeah probably not :-D Using `...` makes it seem like it's somehow a package expression, but it's really referencing a nonexistent file called "...". I'd be fine with something obviously non-existent too like "//:foobar" or "//:dummy"